server/src/test/java/org/elasticsearch/common/compress/DeflateCompressTests.java

```diff
@@ -45,34 +44,17 @@ public void testRandom() throws IOException {
     }
 
     public void testRandomThreads() throws Exception {
-        final Random r = random();
```

Comment:
Made some small adjustments here by moving all the random stuff to our standard utility methods to make things work (can't share `Random` across threads).